src/lib/serviceRoutinesV2/postSubscriptions.cpp

```diff
@@ -62,7 +62,48 @@ extern std::string postSubscriptions
     return answer;
   }
 
-  TIMED_MONGO(ciP->httpStatusCode = mongoSubscribeContext(&parseDataP->scr.res, &scr, ciP->tenant, ciP->uriParam, ciP->httpHeaders.xauthToken, ciP->servicePathV));
+
+  //
+  // String filter in URI param 'q' ?
+  // What if it was also in NotifyConditionVector::NotifyCondition::Expression ... ?
+  //
+  if (ciP->uriParam[URI_PARAM_Q] != "")
```

Comment:
Not in every place... "POST /v2/op/query" is yet using q= and it shuldn't. However, that is addressed in a separate comment, so this one ends with NTC.